components/AnimationTiming/examples/AnimationTimingExample.swift

```diff
@@ -0,0 +1,162 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import UIKit
+import MaterialComponents
+
+struct k {
+   struct AnimationTime {
+      static let interval: Double = 1.0
+      static let delay: Double = 0.5
+   }
+   struct Sizes {
+      static let topMargin: CGFloat = 16.0
+      static let leftGutter: CGFloat = 16.0
+      static let textOffset: CGFloat = 16.0
+      static let circleSize: CGSize = CGSize(width: 48.0, height: 48.0)
+   }
+   
+}
+
+class AnimationTimingExample: UIViewController {
+
+   fileprivate let scrollView: UIScrollView = UIScrollView()
+   fileprivate let linearView: UIView = UIView()
+   fileprivate let materialEaseInOutView: UIView = UIView()
+   fileprivate let materialEaseOutView: UIView = UIView()
+   fileprivate let materialEaseInView: UIView = UIView()
+
+   
+   override func viewDidLoad() {
+      super.viewDidLoad()
+      // Do any additional setup after loading the view, typically from a nib.
+      
+      view.backgroundColor = UIColor(white: 0.9, alpha: 1.0)
+      title = "Animation Timing"
+      setupExampleViews()
+   }
+   
+   override func viewDidAppear(_ animated: Bool) {
+      super.viewDidAppear(animated)
+      let timeInterval: TimeInterval = 2 * (k.AnimationTime.interval + k.AnimationTime.delay)
+      var _: Timer = Timer.scheduledTimer(timeInterval: timeInterval, target: self, selector: #selector(self.playAnimations), userInfo: nil, repeats: true)
+   }
```

Comment:
I think it's missing a call to playAnimations() just after the Timer initialization.

Reply:
Was the animation not playing for you?

Reply:
It was, but instead of starting immediately like the Objective-C example, it was waiting one timer interval to start.